src/index.ts

```diff
@@ -82,6 +86,20 @@ export default function supernova(env: Galaxy) {
       const announce = useAnnounceAndTranslations(rootElement, translator);
       const changeSortOrder = useSorting(model);
 
+      onContextMenu((menu: Menu, event: any) => {
+        areBasicFeaturesEnabled &&
+          event.target &&
+          menu.addItem({
+            translation: 'contextMenu.copyCellValue',
+            icon: 'lui-icon lui-icon--copy',
+            tid: 'copy-cell-context-item',
+            select: async () => {
+              copyCellValue(event.target.firstChild.textContent);
+            },
+          });
+        return Promise.resolve(menu);
```

Comment:
doesn't this make the function async

Reply:
Well, I think(if I'm not wrong) it is not async here because I'm not calling another function after returning the Promise, but then the main function in sense-client that I'm calling `onContextMenu ` in it is an async function `async function getGridCellContextMenu(...)`, does it make sense ?